tests/posix/spinlock/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(posix_spinlock)
```

To match the option group name
```suggestion
project(posix_spinlocks)
```